docs/en/reference/dql-doctrine-query-language.rst

```diff
@@ -1299,7 +1299,7 @@ pass parameters to the query the following methods can be used:
 
 -  ``AbstractQuery::setParameter($param, $value)`` - Set the
    numerical or named wildcard to the given value.
--  ``AbstractQuery::setParameters(array $params)`` - Set an array
+-  ``AbstractQuery::setParameters(ArrayCollection $params)`` - Set an array
    of parameter key-value pairs.
```

Comment:
Description is misleading now, not sure if it's needed now. It's either no description, or something along the lines of "Set a collection of `\Doctrine\ORM\Query\Parameter`". But that's just repeating what's supposed to be obvious from method docblock.

Reply:
I did "Set a `\Doctrine\Common\Collections\ArrayCollection` of `\Doctrine\ORM\Query\Parameter`s."

is that what you want?